apps/src/templates/sectionProgress/sectionProgressLoader.js

```diff
@@ -142,63 +106,29 @@ function processStudentTimestamps(timestamps) {
   return studentTimestamps;
 }
 
-function processStudentPairing(students) {
-  const studentPairing = getStudentPairing(students);
-  const isValid = Object.keys(studentPairing).every(userId =>
-    Object.keys(studentPairing[userId]).every(
-      levelId => typeof studentPairing[userId][levelId] === 'boolean'
-    )
-  );
-  if (!isValid) {
-    throw new Error('Input is invalid');
-  }
-  return studentPairing;
-}
-
 function postProcessDataByScript(scriptData) {
-  if (!scriptData.stages) {
-    return scriptData;
+  // Filter to match scriptDataPropType
+  const filteredScriptData = {
+    id: scriptData.id,
+    csf: scriptData.csf,
+    hasStandards: scriptData.hasStandards,
+    title: scriptData.title,
+    path: scriptData.path,
+    stages: scriptData.lessons,
+    family_name: scriptData.family_name,
+    version_year: scriptData.version_year,
+    name: scriptData.name
+  };
+  if (!filteredScriptData.stages) {
+    return filteredScriptData;
   }
   return {
-    ...scriptData,
-    stages: scriptData.stages.map(stage => {
+    ...filteredScriptData,
+    stages: filteredScriptData.stages.map(stage => {
       return {
         ...stage,
         levels: stage.levels.map(level => processedLevel(level))
       };
     })
   };
 }
```

Comment:
I just had to take a second and point out the EOF. 134 vs. 204. Waaaaaayyyy simpler logic :)